modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -3610,6 +3600,13 @@ private void _insertIntoLocalizationTable(
 
 			FinderCacheUtil.clearDSLQueryCache(
 				dynamicObjectDefinitionLocalizationTable.getTableName());
+
+			for (ObjectField objectField : objectFields) {
+				_putLocalizedValues(
```

Comment:
@MarinhoFeliphe Can we put this inside the upper "for" loop (the one from row 3582) ? And use the same `_getLocalizedValue` method?

Reply:
@gabrielwas I believe it is possible but the code will get more complex because I have to update the values taking the locales into account whereas in this approach I just copy the values to the inserted values map after they are inserted, I think this way is easier to maintain but if you prefer I can try to do this.

Reply:
@gabrielwas there are many changes in the tests but I've only divided the testAddObjectEntry into two (`testAddObjectEntryWithStaticValues` and `testAddObjectEntryWithDynamicValues`)